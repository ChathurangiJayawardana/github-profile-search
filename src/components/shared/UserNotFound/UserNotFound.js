import React from 'react';
import styled from 'styled-components';

import colors from '../../../setup/colors';

const Message = styled.p`
  font-family: 'Raleway';
  font-size: 40px;
  color: ${colors.purple};
`;

export default () => <Message>User Not Found :(</Message>;

