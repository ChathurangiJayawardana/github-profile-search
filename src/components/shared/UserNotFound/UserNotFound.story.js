import React from 'react';
import { storiesOf } from '@storybook/react';
import UserNotFound from '.';

const stories = storiesOf('components/UserNotFound', module);

stories
  .add('default', () => <UserNotFound />);

