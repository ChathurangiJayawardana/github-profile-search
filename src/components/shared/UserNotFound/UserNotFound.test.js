import React from 'react';
import { mount, shallow } from 'enzyme';

import UserNotFound from '.';

describe('UserNotFound Component', () => {
  describe('renders', () => {
    it('render without props ', () => {
      const wrapper = shallow(<UserNotFound />);
      expect(wrapper).toMatchSnapshot();
    });
  });

  describe('content', () => {
    it('should contain 1 paragraph', () => {
      const wrapper = mount(<UserNotFound />);

      expect(wrapper.find('p')).toHaveLength(1);
    });
  });
});

