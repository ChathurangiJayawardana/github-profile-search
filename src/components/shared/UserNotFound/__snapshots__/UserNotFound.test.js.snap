// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserNotFound Component renders render without props  1`] = `ShallowWrapper {}`;
